src/NTH.Windows.Forms/NTH.Windows.Forms/NthListViewGroup.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace NTH.Windows.Forms
+{
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+    public struct NthListViewGroup
```

Comment:
Should this be public? If yes, then there shouldn't be any public fields (use getters/setter instead). If the order of the fields is important for p/invoke reasons, consider creating a wrapping type that is CLS compliant and can be exposed as public api.